testPlugin/testPlugin.go

```diff
@@ -0,0 +1,23 @@
+package main
+
```

Comment:
Spacing needs to be fixed in this file too.

Reply:
fixed